spec/models/user_spec.rb

```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
+  pending "add some examples to (or delete) #{__FILE__}"
+end
```

Comment:
- [ ]  From the model, we can test the association you did in the database configuration, kindly test the association between the tables. For example: `it { should belong_to(:recipe) }`. Please make sure you have tested all the associations.
- [ ] Implement the tests for the model